packages/office-ui-fabric-react/src/components/Button/docs/ButtonA11y.md

```diff
@@ -0,0 +1,12 @@
+Button is accessible if these considerations are followed:
+
+- Use a touch target area of `44px` by `44px`
+- For multiple buttons provided on the page that perform the same function, one of the targets should be `44px` by `44px`.
+- Buttons should get activated using `Enter` key as well as `SPACE` key.
+- For custom buttons made using `<div>`, `<span>`, `<svg>`or `<img>`,
+  - `role="button"` must be applied.
+  - Tabindex property should be set to "0", so that it could get tab focus.
```

Comment:
This is wrong guidance.  We should not be asking users to worry about tabIndex, when the component already manages this for you.

We need to separate 2 concerns:

1. Things the Button will manage for you, so that you (the user) don't need to. 

You don't need to worry about tabbing or spacing. You don't need to worry about tab index. You don't need to worry about "custom buttons" unless you're dealing with an `as` prop, and even then maybe WE should be providing role.button and not even asking the user to deal with it.

2. Things that you (the user) need to consider and HOW. 

For example, perhaps tooltips are a good idea for icon buttons. How? What's the recommended accessible approach? Just adding `title` would do it, but you wouldn't get a consistent look/feel. Perhaps we should add a custom tooltip here. We need a code example.

Reply:
Just saw @ecraig12345 comment, which is effectively the same thing i just said :)